### DISCUSSION TITLE
ctx.guild.fetch_member() hangs if the user has left the server

### DISCUSSION THREAD

**Discussion by tlk3** [Pycord-Development/pycord#1969] [Q&A] [answered]
Has anyone else experienced this issue? Nothing is returned, so this doesn't work:
     
`member = await ctx.guild.fetch_member(1234567890)`
`if not member: continue`   

I'd rather not query the entire list of server users in advance and filter. Feedback would be greatly appreciated.

**Comment by JustaSqu1d** [comment, marked as answer]
Please use the [support server](https://pycord.dev/discord) for coding help.

If you believe there is a bug with the library please create an issue.

**Reply by tlk3** [reply]
TY. This just for general discussion? I thought it was strange there was no bug report option.
